aoe4/formatters.js

```diff
@@ -124,13 +129,20 @@ class NightbotDefaultFormatter {
 
     const rank = mode.rank ? `rank #${mode.rank}` : 'unranked';
     const rank_level = mode.rank_level ? formatRankLevel(mode.rank_level) + ', ' : '';
-    var msg = `${player.name} is ${rank} (${rank_level}${mode.rating} Elo), with ${mode.games_count} game${mode.games_count == 1?'':'s'} (${mode.wins_count}-${mode.losses_count} | ${mode.win_rate}%)`;
+    const rating = player.rating ? ` (${rank_level}${mode.rating} Elo)` : '';
+    const games = mode.games_count ? mode.games_count : "no";
```

Comment:
Is there a reason to replace '0'  with 'no'?

PS: you can do `mode.games_count || 0`  or `mode.games_count || 'no'` to handle this without ternary ?: expression.

Reply:
I know I can use `||` in other languages, wasn't sure about JS. This appeared in the suggested result in the issue (same as line 134).